venues/ML_Reproducibility_Challenge/2020/python/load_papers.py

```diff
@@ -38,7 +38,7 @@
 
 def post_paper(paper, client):
     authors = ""
-    if not math.isnan(paper['Authors']):
+    if paper['Authors']:
```

Comment:
I think the papers for which we got the error were venue names. Do we want to add the venues as accepted papers?